src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // перетворити рядок на масив де елемент це правило css
+  // перевірити чи рядок не порожній
+
+  const listStyles = sourceString
+    .split(';')
+    .filter((string) => string.trim() !== '');
+
+  const stylesObject = {};
+
+  listStyles.forEach((entry) => {
```

Comment:
The variable naming `entry` is clear, but you could consider a more descriptive name such as `styleRule` or `styleDeclaration` to enhance readability.